demisto_sdk/__main__.py

```diff
@@ -933,11 +933,15 @@ def json_to_outputs_command(**kwargs):
 @click.option(
     '-o', '--output',
     required=False,
-    help='Specify output directory')
+    help='Specify output directory or path. '
```

Comment:
Because you also support specific file input, please add here in help that if path to a specific YML is given what will be the result.